server/server.go

```diff
@@ -242,10 +248,15 @@ func (c *Server) setupRouter(ctx context.Context, logger *slog.Logger) (context.
 func (c *Server) ListenAndServe(ctx context.Context, logger *slog.Logger) error {
 	_, srv := c.setupRouter(ctx, logger)
 
+	ServeMetrics()
```

Comment:
Will this be called twice once [here](https://github.com/brave-intl/challenge-bypass-server/pull/809/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R89) and then again [here](https://github.com/brave-intl/challenge-bypass-server/pull/809/files#diff-2873f79a86c0d8b3335cd7731b0ecf7dd4301eb19a82ef7a1cba7589b5252261R94) when staring the server. 

If so could it just be started once with the main server,